inject.js

```diff
@@ -49,13 +103,26 @@ chrome.runtime.onMessage.addListener(function(site) {
   function done() {
     if (span.textContent === '') {
       chrome.storage.sync.remove(uuid);
-      span.textContent = titles[uuid];
+      span.textContent = title;
```

Comment:
This introduces a regression. The reason I had the titles object was to keep track of the *original* name of the scholar site (as known on initial page load) so that when you removed the name it would revert to scholar's actual name, rather than the name it was known as before you tried to edit it this time.

Also, in the new semantics you introduce, an empty name will revert it to the previously-known user-chosen name in the current loaded instance of the page, but still removes it from the chrome storage backend thing, so the next time you load the page the name will be the original scholar name (different from how you left it). This violates the rule of least surprise.